### DISCUSSION TITLE
Trying to make a database, not an issue but please read.

### DISCUSSION THREAD

**Discussion by TheLastRavenpuff** [nirgoren/Eddienput#34] [Scripts and Configs]
Ok I wasn't sure how to reach you so here am. I'm wanted to make a databank of sorts for this project where people could upload and share scripts for easy community access. https://drive.google.com/drive/folders/19mDfjmVRLwYUjt31pHtGNoYta1KLjhPf?usp=sharing
This is the link, it is currently unfinished cause its late where I live at the time of posting this but I'll get it fully up and running tommorow. When finished it'll be organized game>character>combo/tech/mixup. If you like this idea please post it somewhere, preferably on this project itself so we can start populating it and making everyone's life easier by having scripts all ready for them, and if it doesn't exist they can make it themselves and add to this folder for the next person. Thank you for reading!

**Comment by KuukuuOmoi** [comment]
currently we have an unofficial discord as well to discuss over some of the things that come up as well. 

https://discord.gg/xTMaXjqQ
